test/jdk/ProblemList-FIPS140_2.txt

```diff
@@ -1238,3 +1238,5 @@ java/util/jar/JarFile/IgnoreUnrelatedSignatureFiles.java    https://github.ibm.c
 # https://github.ibm.com/runtimes/backlog/issues/1089
 javax/security/auth/kerberos/StandardNames.java https://github.ibm.com/runtimes/backlog/issues/1089 linux-x64,linux-ppc64le,linux-s390x
 sun/security/krb5/auto/CaseSensitive.java https://github.ibm.com/runtimes/backlog/issues/1089 linux-x64,linux-ppc64le,linux-s390x
+
+javax/rmi/ssl/SocketFactoryTest.java https://github.com/eclipse-openj9/openj9/issues/21756 linux-x64,linux-ppc64le,linux-s390x
```

Comment:
Please arrange the new entries in this file and other two files in alphabetical order.

Reply:
I think the items in the same group should be in alphabetical order.

Reply:
Insert each item into the appropriate list in alphabetical order.